wa/workloads/speedometer/__init__.py

```diff
@@ -120,6 +120,11 @@ class Speedometer(Workload):
                   The app package for the browser that will be launched.
                   """,
         ),
+        Parameter(
+            'version',
+            kind=str, default="123",
+            description='Specifies which version of the workload should be run.',
+            override=True),
```

Comment:
A `ApkUiautoWorkload` base class should already define a `version` Parameter, is there a specific reason this needs to be overridden?

Reply:
Hi Marc,
you're right. I missed that. I deleted the `version` parameter. 
Instead I have added `prefer_host_package` and put default as `false` to avoid WA tries to install the version on the host.